### PR TITLE
Use multiselects for the recipient selection

### PR DESCRIPTION
**Pull request by ChristophWurst**
@jancborchardt do you have a few minutes to check this out and polish this a bit? I've already adjusted this a lot to look *almost* like the previous layout/design, but a few pixels still have to be moved around.

- [x] Restructuring from input -> multiselect
- [x] Fix layout (the part where I need help from 🧙‍♀️🧙‍♂️🧙)
- [x] Query autocomplete addresses from server and add to Vuex store for the multiselects to query
- [x] Make sure this works for `to`
- [x] Make sure this works for `cc`
- [x] Make sure this works for `bcc`
- [x] Reduce recipient array to string on send
- [x] Convert string recpients from `mailto:` into label-email objects
- [x] Make sure groups still work as recipients

### CODE REVIEW

**Comment by jancborchardt** [comment]
@ChristophWurst sorry, am totally dead from working for the feature freeze. :O (It’s 5:16 in the morning here for example.) And got a conference + travel coming up this weekend.

Can someone else from @nextcloud/designers help out! :heart_eyes:

**Comment by ChristophWurst** [comment]
> @ChristophWurst sorry, am totally dead from working for the feature freeze. :O (It’s 5:16 in the morning here for example.) And got a conference + travel coming up this weekend.

No worries :)

**Comment by ChristophWurst** [comment]
>     * Fix layout (the part where I need help from 🧙‍♀️🧙‍♂️🧙)

@skjnldsv this looks like a job for you :wink:

**Comment by skjnldsv** [comment]
capture d ecran_2018-11-14_10-30-14

**Comment by skjnldsv** [comment]
> @skjnldsv what's the status here? :)

Well, I was pretty much done :see_no_evil:

**Comment by ChristophWurst** [comment]
I didn't know :man_shrugging: 

Awesome! Will try to continue with this then. Thanks a lot :v:

File: src/components/Address.vue [hunk 1]
```diff
@@ -1,6 +1,14 @@
 <template>
-	<span :title="email"
-		  v-tooltip.bottom="label">{{ label }}</span>
+	<router-link :to="{name: 'message', params: {
+								accountId: $route.params.accountId,
```
Comment (skjnldsv) [inline src/components/Address.vue]: Too much tabs? :p

Comment (ChristophWurst) [reply src/components/Address.vue]: bildschirmfoto von 2018-11-21 18-47-43
works on my machine :stuck_out_tongue:

Comment (ChristophWurst) [reply src/components/Address.vue]: I should move that to a computed prop. it's ugly in the template.

Comment (skjnldsv) [reply src/components/Address.vue]: > works on my machine :stuck_out_tongue:

We have very different coding style ^^'

Comment (ChristophWurst) [reply src/components/Address.vue]: Yeah. Please use mine, it's so much better :wink:

**Pull request merged**